djangocms_navigation/admin.py

```diff
@@ -24,6 +28,11 @@
 from .views import ContentObjectSelect2View, MenuContentPreviewView
 
 
+IS_POPUP_VAR = '_popup'
```

Comment:
This could be imported so no need to redefine the var

from django.contrib.admin.options import IS_POPUP_VAR

https://github.com/django/django/blob/main/django/contrib/admin/options.py#L70